contrib/sbase/patches/creat2open.patch

```diff
@@ -0,0 +1,51 @@
+Index: sbase/sbase/libutil/cp.c
+===================================================================
+--- sbase.orig/sbase/libutil/cp.c	2022-12-06 19:10:45.123093136 +0100
++++ sbase/sbase/libutil/cp.c	2022-12-07 22:19:23.039798679 +0100
+@@ -111,14 +111,14 @@
+ 			cp_status = 1;
+ 			return 0;
+ 		}
+-		if ((f2 = creat(s2, st.st_mode)) < 0 && cp_fflag) {
++		if ((f2 = open(s2, O_CREAT|O_WRONLY|O_TRUNC, st.st_mode)) < 0 && cp_fflag) {
```

Comment:
Please add `creat` wrapper for `open` in exactly the same way as NetBSD did that in:
https://mimiker.ii.uni.wroc.pl/source/xref/NetBSD/lib/libc/compat-43/creat.c?r=53d22107#46